### ISSUE TITLE
In policy reform calculator, policies are automatically named with just a number

### ISSUE THREAD

**Issue by anth-volk** [PolicyEngine/policyengine-app#886]
For instance, when I abolished above-the-line deductions, the auto-generated name of the policy was just "6456." Is this normal behavior? I thought I remembered some text added before the number.

**Comment by MaxGhenis** [comment]
Yes, we name policies `Policy #x` by default. That's what I'm seeing for 6456 too:

https://policyengine.org/us/policy?focus=gov&reform=6456&region=us&timePeriod=2023&baseline=2

<img width="443" alt="image" src="https://github.com/PolicyEngine/policyengine-app/assets/6076111/9f341123-eb48-4071-97bc-6ae1f8ccd740">

Are you looking at a different policy?

**Comment by anth-volk** [comment]
I’m sorry, I wasn’t clear. What loaded for me was just the “#6456” part, without “Policy” in front. That said, when I open the link you shared, I receive it with the “Policy” in front, so I have no idea what happened.
